### ISSUE TITLE
YAML Editing: Snippets, Context not working

### ISSUE THREAD

**Issue by Janfy** [GoogleCloudPlatform/cloud-code-vscode#115]
**Environment**:
*(Hint: "Report Kubernetes Extension Issue on Github" command will fill these out for you.)*
Extension version: 0.0.9
VSCode version: 1.35.0
OS: 10.0.17134

**Description**:

When editing yaml file, Kubernetes snippet are not available, neither context completion.

I've enable **verbose** for Trace Server setting, But the only error i have is :

```
[Error - 10:49:05] glob error
```

Hints:

- I need to ue proxy erver (with authentication) to access the internet
- Works as expected on other computer, with direct access to internet

**Repro step**:

Create new file named `test.yaml`, save it. Press Ctrl + Space -> No suggestions

**Comment by redborian** [comment]
Thanks for providing more information @Janfy. The suggested naming convention is to end file names with `.yaml`. That should fix your issue. That being said, we will make sure we support other file types as well in future releases.

**Comment by Janfy** [comment]
It's a mistake (i correct it) in the issue. My test file is corretly named `test.yaml` with the `.yaml` extension and not `.yml`.

**Comment by Janfy** [comment]
The error in console log:
`[Error - 10:49:05] glob error`
appears a few seconds after starting VsCode, with no open file in the editor.

**Comment by redborian** [comment]
Thanks for the clarification @Janfy. Are you able to run kubectl commands out of the box or do you need authentication for it as well? Are other features (like hover, schema validation) working well for you?

**Comment by Janfy** [comment]
kubectl commands works out of the box.
The kubernetes explorer works fine (slowly, but it works).
No others feature is working when i got the error `glob error`.

But i think i found how to reproduce this error.

I've got this error when the project [Kubespray](https://github.com/kubernetes-sigs/kubespray) is added to a workspace.

* Close all files,
* Close all workspaces
* Clone [Kubespray](https://github.com/kubernetes-sigs/kubespray)
* Restart VSCode
* Add Folder `kubespray` to a new workspace

And after few seconds, the glob error appears in output window.

When editing a .yaml file whitout any workspace open, code completion, hover et schema validation are working as expected!

**Comment by Janfy** [comment]
I've tried to create a new workspace with the follwing folders:

```
.
├── conf-k8s 
└── helm-repository
```
`conf-k8s`  contains few .yaml files to override `values.yaml` from the `helm-repository`, which contains some charts forked from the official repository.

Everything works fine until i had the `kubespray` folder to the workspace.

After a VsCode restart, i've the following error message:

```
[Error - 08:27:17] Server initialization failed.
  Message: Request initialize failed with message: ENOENT: no such file or directory, open 'c:\gR\forge\helm-repository\jenkins\preprod.yaml'
  Code: -32603 
```
And after this error, completion, hover and schema validation are not working anymore.

The file `jenkins\preprod.yaml` is missing because it belongs to the `conf-k8s` directory, not the `helm-repository`! Very strange behaviour!

For the moment, my workaround is to **not add** `kubespray` folder to my workspace.

**Comment by quoctruong** [comment]
@redborian Is this related to https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/108?

**Comment by sujit-kamireddy** [comment]
This issue has been fixed in the latest release 0.0.10. Please try it out.

As always, please keep issues coming our way.
